Doppler.Integrations/Controllers/TypeformController.cs

```diff
@@ -60,7 +60,11 @@ public async Task<IActionResult> AddSubscriber(string accountName, long idList,
             catch (Exception ex)
             {
                 _log.LogError(new EventId(), ex, string.Format("AccountName: {0} | ID_List: {1} | Status: Add subscriber has failed", accountName, idList));
-                return new BadRequestResult();
+                return BadRequest(new
+                {
+                    ErrorMessage = "An subscriber email must be provided",
```

Comment:
We talked about this, at this instance a number of other things aside from email could throw an exception so this message is not accurate, in ErrorMessage I'd put the exception message, and also a field with the StackTrace